src/components/navbar/Navbar.js

```diff
@@ -18,10 +18,10 @@ export default class Navbar extends Component {
                 </div></a>  
             <a href='#events'><div className='codeutsava__navbar-menu-heading'>
                     Events
-                </div></a>  
-                <div className='codeutsava__navbar-menu-heading'>
-                    Sponsors
-                </div>
+                </div></a>
```

Comment:
Instead of removing the "sponsors" section, add another section named "speakers" after the sponsors one.

Reply:
added both "sponsors" and "speakers sections. added "speakers" section after the "sponsors" section in navbar .